src/Components/Aspire.Confluent.Kafka/KafkaProducerSettings.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Confluent.Kafka;
+
+namespace Aspire.Confluent.Kafka;
+
+public sealed class KafkaProducerSettings
+{
+    public string? ConnectionString { get; set; }
+
+    public ProducerConfig? Config { get; internal set; }
+
+    public bool Metrics { get; set; } = true;
+
+    public bool HealthChecks { get; set; } = true;
+
+    internal void Consolidate()
+    {
+        Debug.Assert(Config is not null);
+
+        if (ConnectionString is not null)
+        {
+            Config!.BootstrapServers = ConnectionString;
```

Comment:
```suggestion
            Config.BootstrapServers = ConnectionString;
```

You've already asserted it is not null. (applies in other places as well)

Reply:
done